static/js/composebox_typeahead.js

```diff
@@ -578,7 +578,15 @@ exports.get_sorted_filtered_items = function (query) {
             big_results.is_silent, token, opts);
     }
 
-    return exports.filter_and_sort_candidates(completing, big_results, token);
+    const result = exports.filter_and_sort_candidates(completing, big_results, token);
+
+    // When the user types #query we want to also
+    // suggests recent topics that matches with the query.
+    if (completing === 'stream') {
+        const topics_result = match_stream_query_with_recent_topics(query.substring(1));
```

Comment:
If you return here, are you skipping the filtering/sorting step?

Reply:
At this point, I already have the stream results filtered and sorted in 'cost result'. I just match the recent topics with the query and concatenate that with the stream result. (I'm not sure if I understood your question right, please clarify if I didn't answer what you asked).